ui/page/collection/internal/collectionActions/view.jsx

```diff
@@ -0,0 +1,110 @@
+// @flow
+import * as ICONS from 'constants/icons';
+import React from 'react';
+import Button from 'component/button';
+import { useIsMobile } from 'effects/use-screensize';
+import ClaimSupportButton from 'component/claimSupportButton';
+import ClaimShareButton from 'component/claimShareButton';
+import FileReactions from 'component/fileReactions';
+import classnames from 'classnames';
+import { ENABLE_FILE_REACTIONS } from 'config';
+import ClaimRepostButton from 'component/claimRepostButton';
+import PlayButton from './internal/playButton';
+import ShuffleButton from './internal/shuffleButton';
+import CollectionDeleteButton from './internal/deleteButton';
+import CollectionPublishButton from './internal/publishButton';
+import CollectionReportButton from './internal/report-button';
+
+type Props = {
+  uri: string,
+  claimId?: string,
+  isMyCollection: boolean,
+  collectionId: string,
+  showInfo: boolean,
+  setShowInfo: (boolean) => void,
+  showEdit: boolean,
+  setShowEdit: (boolean) => void,
+  isBuiltin: boolean,
+  collectionEmpty: boolean,
+};
+
+function CollectionActions(props: Props) {
+  const {
+    uri,
+    claimId,
+    isMyCollection,
+    collectionId,
+    showInfo,
+    setShowInfo,
+    isBuiltin,
+    showEdit,
+    setShowEdit,
+    collectionEmpty,
+  } = props;
+
+  const isMobile = useIsMobile();
+
+  return (
+    <div className={classnames('media__actions justify-space-between', { stretch: isMobile })}>
+      <SectionElement>
+        {!collectionEmpty && <PlayButton collectionId={collectionId} />}
+        {!collectionEmpty && <ShuffleButton collectionId={collectionId} />}
+
+        {!isBuiltin && (
+          <>
+            {uri && (
+              <>
+                {ENABLE_FILE_REACTIONS && <FileReactions uri={uri} />}
+                <ClaimSupportButton uri={uri} fileAction />
+                <ClaimRepostButton uri={uri} />
+                <ClaimShareButton uri={uri} fileAction webShareable />
```

Comment:
- [ ] :exclamation: `ClaimShareButton` is missing the `collectionId` parameter here (or maybe somewhere else), so the Social Share modal doesn't show the "Include playlist ID" widget.

<img width="482" alt="image" src="https://user-images.githubusercontent.com/64950861/177510245-9b116330-e6af-4194-b4db-c5f60f368cb6.png">